validator.py

```diff
@@ -406,7 +420,9 @@ def validate_dependencies(self):
         self.value_follows_list()
         self.character_matching()
         self.verify_uniqueness()
-        self.end_scene_allowed()
+        self.verify_allowed_actions()
+        # self.end_scene_allowed() # no longer needed
```

Comment:
Probably remove this comment?

Reply:
Left it in case we needed to reinstate, since it seemed unsure what to do with this one. I will remove the line and the function